src/restic/node.go

```diff
@@ -581,6 +581,7 @@ func (node *Node) fillExtra(path string, fi os.FileInfo) error {
 	case "file":
 		node.Size = uint64(stat.size())
 		node.Links = uint64(stat.nlink())
+		node.Device = uint64(stat.dev())
```

Comment:
No, that's not possible: It would change the semantic of the `Device` field, which is set to `st_rdev` right now, that would break existing repos. I'm already working on it, we'll add a new field `DeviceID` for `st_dev`.